code_challenges/code_wars/trigrams.py

```diff
@@ -0,0 +1,19 @@
+# https://www.codewars.com/kata/55d8dc4c8e629e55dc000068
+
+
+def extract_trigrams(words):
+    stopping_point = len(words) - 2
```

Comment:
This is a good pattern, doing some initial math and saving the value in a variable with an applicable name.